pyproject.toml

```diff
@@ -1,11 +1,46 @@
-[build-system]
+[project]
 name = "ElectionLeaflets"
-version = "3.0-alpha"
-requires = [
-    "setuptools >= 35.0.2",
-    "setuptools_scm >= 2.0.0, <3"
+version = "0.1.0"
+description = " 📮 A Django port of the original PHP code from electionleaflets.org "
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "aws-wsgi",
+    "dc-design-system",
+    "dc-django-utils",
+    "dj-pagination==2.4.0",
+    "django-braces==1.16.0",
+    "django-extensions==3.1.5",
+    "django-filter==24.3",
+    "django-formtools==2.2",
+    "django-localflavor==4.0",
+    "django-pipeline==3.1.0",
+    "django-s3file==5.5.7",
+    "django-ses==4.3.0",
+    "django-static-jquery==2.1.4",
+    "django-storages==1.13.2",
+    "django-uk-political-parties",
+    "django==4.2.15",
+    "djangorestframework==3.15.2",
+    "factory-boy==3.3.1",
+    "markdown==3.7",
+    "piexif==1.0.12",
+    "pillow==10.4.0",
+    "psycopg2-binary==2.9.10",
+    "sorl-thumbnail===12.11.0",
+    "setuptools>=75.6.0",
+    "aws-wsgi",
+    "sentry-sdk==2.19.0",
 ]
-build-backend = "setuptools.build_meta"
+
+[tool.uv.workspace]
+members = ["thumbs"]
+
+[tool.uv.sources]
+django-uk-political-parties = { git = "https://github.com/DemocracyClub/django-uk-political-parties.git", rev = "86ffa51f8306858c379c9de5be2f4bdb24b3a910" }
+dc-design-system = { git = "https://github.com/DemocracyClub/design-system.git", rev = "c5c5de6b792b4b57997b65370fdcdfa4f099cdf4" }
+dc-django-utils = { git = "https://github.com/DemocracyClub/dc_django_utils.git", rev = "7.0.1" }
+aws-wsgi = { git = "https://github.com/DemocracyClub/awsgi.git", rev = "b286f129c5580547577f942c75be74cdca707386" }
```

Comment:
What is your thinking on using [dependency sources](https://docs.astral.sh/uv/concepts/projects/dependencies/#dependency-sources) here?
I remember I tried out a few different ways of dealing with VCS dependencies when I first looked at UV and I think I came to the conclusion that just doing it PEP508 style:
https://github.com/DemocracyClub/EveryElection/blob/8e27f3f512badddafcb77e2154a5069af01bfb42/pyproject.toml#L37-L38
is probably the clearest way to do it because aside from anything else it keeps the package and version in one place - same as all the other packages.

Reply:
I left this in by mistake. I moved to them as I thought it might be a source of the SAM bug I fixed later. I've reverted back